src/content/blog/posts/post-1.md

```diff
@@ -1,18 +1,193 @@
+
+# Why store everything in GitHub?
+
+The tools discussed in this article can have less features and may involve a steeper learning curve than your existing setup. In addition, there aren't clear migration paths which means that unless you're starting from scratch it can be painful to move accross.
+
+However, there are various advantages which makes it worthwhile:
+
+### Searching
+
+Gathering all your code, tickets, documentation, and presentations in one place makes it super easy to search through everything, *if you can find it*. However, GitHub Code Search which was first released in [2008](https://github.blog/2021-12-15-a-brief-history-of-code-search-at-github/) used to [suck](https://github.com/isaacs/github/issues/908). GitHub rolled out a new code search engine and browser in [November 2022](https://github.blog/changelog/2022-11-09-introducing-an-all-new-code-search-and-code-browsing-experience/), finally enabling users to find relevant results at reasonable speed and turning GitHub into a viable developer hub.
+
+### Interoperability
+
+Using GitHub tools exclusively makes it easier to link and automate across your project artefacts. Hence it makes it easier to track, review and deploy changes as well as collaborate with your team and users.
+
+### Standardisation
+
+Treating project [deliverables as code](https://thenewstack.io/why-docs-as-code-should-be-part-of-your-dev-cycle/) means you can leverage the same practices that are applied to software development: version control, automation, reusability and maintainability. This leads to improved standardization and quality across your projects and enterprise.
+
+### Cost
+
+You company may save money by no longer needing licenses for redundant tools like JIRA and Atlassian. However, this benefit is less significant if your company still has to pay for licenses for teams that don't use GitHub and are unlikely to switch.
+
+# Project Management
+
+In [August 2022](https://github.blog/2022-07-27-planning-next-to-your-code-github-projects-is-now-generally-available/) Github released the new version of [GitHub Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) which summarises GitHub repository issues and PRs through a more intuitive GUI. Whilst [Projects Classic](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/about-project-boards) were tied to a repository, Projects V2 are created at the GitHub organization level. This means they can be linked to a repository, but are independent of them. You can also add issues from [different GitHub organisations](https://github.blog/changelog/2023-02-23-github-issues-projects-february-23rd-update/#add-cross-organization-issues-and-pull-requests-to-projects), but there won't be a backwards-link on the issue to the project.
```

Comment:
Can we change:
- GUI to user interface
- Projects Classic to classic projects
- Projects V2 to new projects